### PR TITLE
(705) Refactor locale files part three

### PR DESCRIPTION
**Pull request by mec**
## Changes in this PR
It's a trilogy! 😄 This is the draft of part 3 that includes:

- the final move of translations into their new model homes
- a refactoring of the Rails flash messages (5e244bd)

## There's more?!?! 😱 
Yes, I'm afraid so… Next up we'll (finally):

- add the planned new structure for forms, tables, document_titles, etc
- refactor any related helpers
- refactor the govuk form builder
- remove any resulting redundant translations

See you in part four…

### CODE REVIEW

File: config/locales/models/budget.en.yml [hunk 1]
```diff
@@ -34,3 +34,34 @@ en:
     budget:
       edit: Edit budget
       new: Create budget
+  activerecord:
```
Comment (tahb) [inline config/locales/models/budget.en.yml:39]: I've probably said this already but I'd like to say it again: I think having a file per resource like this is excellent. It is contextual and you can see how language about the same resource is used in different places without distraction 👌

File: config/locales/models_and_forms.en.yml [hunk 2]
```diff
@@ -1,6 +0,0 @@
----
```
Comment (tahb) [inline config/locales/models_and_forms.en.yml:1]: 🙌 Bye bye!

File: app/controllers/staff/extending_organisations_controller.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@ def update
 
     if @activity.valid?(:update_extending_organisation)
       @activity.save
-      flash[:notice] = I18n.t("form.activity.update.success")
+      flash[:notice] = I18n.t("action.#{@activity.level}.update.success")
```
Comment (tahb) [inline app/controllers/staff/extending_organisations_controller.rb:17]: Good catch 👍

**Comment by mec** [comment]
[Refactor our internationalisation files](https://trello.com/c/7iYw0d3B/705-refactor-our-internationalisation-files)

**Pull request merged**